spec/services/git_based_domain_import_service_spec.rb

```diff
@@ -1,66 +1,87 @@
 describe GitBasedDomainImportService do
-  describe "#import" do
-    let(:git_repo) { double("GitRepository", :git_branches => git_branches, :id => 123) }
-    let(:domain) { double("MiqAeDomain") }
+  let(:git_repo) { double("GitRepository", :git_branches => git_branches, :id => 123) }
+  let(:domain) { double("MiqAeDomain") }
+  let(:user) { FactoryGirl.create(:user) }
```

Comment:
There's no real reason to create a user here, the only thing you need it for is the userid, which you can give a canned response for. Or, maybe we should think about passing the userid directly from the controller that uses this service so we don't have to worry about a `User` in here at all?